__tests__/app.test.js

```diff
@@ -244,6 +244,30 @@ describe("PATCH /api/articles/:article_id", () => {
     });
 })
 
+describe.only("DELETE /api/comments/:comment_id", () => {
```

Comment:
You should remove the .only and check that this does not interfere with any other tests